.github/workflows/linter.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
```

Comment:
Should it be `Lint` here? And the workflow's name & file name should be a verb following our convention 😄

Reply:
Updated https://github.com/Lahphim/react-surveyor-challenge/pull/47/commits/56ef115b264e6bec6b973876dfcf3acb387db107